README.md

```diff
@@ -456,23 +456,22 @@ ConfigureMyObject(
     downstreamDependencies: originalObjectDependencies.Where(x => x.Direction == Direction.Downstream));
 ```
 
-### [F.3.4] ✔️ **CONSIDER** Splitting long chains of dotted method/property invocations
+### [F.3.4] ✔️ **DO** Split chains of method invocations with lambda expressions 💻
 
-If you invoke a chain of methods and/or properties and that single line becomes very long, split each invocation onto its own line. Use well-named local variables if needed.
+If you invoke a chain of methods with lambda expressions, split each invocation (except the first one) onto its own line. Use well-named local variables if needed.
```

Comment:
@gregr-ni what are your thoughts on this update, specifically the split each invocation (except the first one)? That's how it's currently implemented, but we can easily change that. On one hand, I don't like leaving the first on the first line similar to how we treat parameters in methods, but on the other, I don't love putting each on their own line, since it will potentially end up with extremely short lines of code:

```csharp
var disctinctSelectedSoftware = selectedSoftwareContents.GroupBy(x => x.AliasName)
    .Select(x => x.First());
```
vs
```csharp
var disctinctSelectedSoftware = selectedSoftwareContents
    .GroupBy(x => x.AliasName)
    .Select(x => x.First());
```

Reply:
I want to get @ccifra 's thoughts as well

Reply:
I think either should be allowed and the fixer can either leave the first on the same line or use a length heuristic. I would not want the fixer to always go to a second line.

Reply:
@gregr-ni do you think we need to change the wording in this PR? Currently it says (except the first one) but I don't know if it's necessary to change that.

Reply:
I'm ok with the wording as is.